### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated the docs contribution information to include links to product docs, and DWG channel. Intention is to move contributions into the new doc set.

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # Focalboard website
 
-This folder is used to manage the website at [focalboard.com](https://www.focalboard.com/). It was built using [Hugo](https://gohugo.io/).
+This folder is used to manage the website at [focalboard.com](https://www.focalboard.com/). It was built using [Hugo](https://gohugo.io/). The website
```
Comment (wiggin77) [inline website/README.md:3]: Sentence fragment dangling at the end there.

**Review by wiggin77 — CHANGES_REQUESTED** [review]

File: website/README.md (hunk 1 above)
Comment (justinegeffen) [reply website/README.md]: Thank you for picking this up!

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by hahmadia — APPROVED** [review]

File: website/README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ This folder is used to manage the website at [focalboard.com](https://www.focalb
 
 ## How to contribute
 
-You're welcome to submit pull requests for typos, minor corrects and additions, troubleshooting tips to install guides and other incremental improvements.
+The documentation for Focalboard (now known as Boards) has moved to the [Mattermost Product Documentation](https://docs.mattermost.com/guides/boards.html) site.
```
Comment (chenilim) [inline website/README.md:10]: We should clarify that "Boards" is the name in Mattermost, e.g.:
> The documentation for Focalboard (also known as Boards in Mattermost) has...

**Review by chenilim — CHANGES_REQUESTED** [review]
Sorry for the delay. I realize it's a bit vague, but for a Focalboard-audience, we should be careful to note that "Boards" is the name only when part of Mattermost - it's Focalboard for Personal Edition users.

**Pull request merged**